libs/dal/src/repositories/integration/integration.repository.ts

```diff
@@ -34,6 +34,10 @@ export class IntegrationRepository extends BaseRepository<IntegrationEntity> {
       channel: data.channel,
     });
     if (test.length === 0) {
+      if (data.active === true) {
+        await this.deactivatedOtherActiveChannels(data);
+      }
```

Comment:
I've addressed this in the previous #334 PR, let me know what you think

Reply:
I agreed on the matter, in addition, this branch is a bit behind we need to merge the new changes into it